rust/tw_bitcoin/src/modules/legacy/scripts.rs

```diff
@@ -0,0 +1,249 @@
+use crate::aliases::*;
+use bitcoin::PublicKey;
+use std::ffi::{c_char, CStr};
+use tw_memory::ffi::c_byte_array::CByteArray;
+use tw_memory::ffi::c_byte_array_ref::CByteArrayRef;
+use tw_misc::try_or_else;
+use tw_proto::Bitcoin::Proto as LegacyProto;
+use tw_proto::BitcoinV2::Proto;
+
+#[no_mangle]
+#[deprecated]
+// Builds the P2PKH scriptPubkey.
+pub unsafe extern "C" fn tw_build_p2pkh_script(
```

Comment:
Could you please move these functions to either `wallet_core_rs/src/ffi/` or `tw_bitcoin/src/ffi/`?